cloud/kubernetes/README.md

```diff
@@ -71,15 +71,6 @@ first node is special in that the administrator must manually prepopulate the
 parameter. If this is not done, the first node will bootstrap a new cluster,
 which will lead to a lot of trouble.
 
-### Secure mode
-
-Secure mode currently works by requesting node/client certificates from the kubernetes
```

Comment:
It might be good to keep this part, it's still true. The only change is that restarted pods don't need to get a new CSR, they will use the certificate and key stored in secrets.

Reply:
K, added back in. I didn't understand that our current approach overall is a "limitation".